Formula/jfrog-cli.rb

```diff
@@ -17,14 +18,17 @@ class JfrogCli < Formula
   depends_on "go" => :build
 
   def install
-    system "go", "build", "-ldflags", "-s -w -extldflags '-static'", "-trimpath", "-o", bin/"jfrog"
+    system "go", "build", *std_go_args(ldflags: "-s -w -extldflags '-static'", output: bin/"jf")
     prefix.install_metafiles
```

Comment:
The reference to metafiles can be removed

Reply:
Removed. Do you know what it does?

Reply:
Installs the `README`, `LICENSE`, etc. Older versions of Homebrew didn't do it automatically.